pkg/client/opts.go

```diff
@@ -88,11 +90,21 @@ func WithBearer(token *token.BearerToken) CallOption {
 	}
 }
 
+// WithSidechainMagic returns option to specify network magic of NeoFS sidechain.
+//
+// Zero by default.
+func WithSidechainMagic(magic uint64) CallOption {
```

Comment:
Wouldn't it be quite a pain to provide `clients.WithSidechainMagic` in every request we send in neofs-node? Maybe we will benefit from `WithDefaultSideChainMagic` client option as well?

Reply:
I made it a `CallOption` after thought "oh, magic is a request field, so it is a per-call value". But in current implementation we expect that storage node can't switch magic number within opened connection, so we can change it to `Option`.